src/Components/signup/Signup.jsx

```diff
@@ -13,42 +13,39 @@ import TextField from '@material-ui/core/TextField';
 
 const useStyles = makeStyles((theme) => ({
   avatar: {
-    marginTop: '10px',
+    margin: '10px auto',
     backgroundColor: 'green',
   },
   paper: {
     width: '50vw',
-    margin: 'auto',
-    marginBottom: '3rem',
-    height: '100vh',
+    margin: '1rem auto 3rem auto',
+    padding: '2rem',
     display: 'flex',
     flexDirection: 'column',
+    backgroundColor: 'rgba(255, 255, 255, 0.9)',
```

Comment:
Can you define these colors and grandients in the theme component, like color1, color2,....and then import them here?
Do these for all the files wherever colors are involved(navbar, forms, footer, heading text color, paragraph text color, etc)